### DISCUSSION TITLE
I got a non-iframeable website (YouTube) to work. I can even watch videos.

### DISCUSSION THREAD

**Discussion by Pixel556** [DustinBrett/daedalOS#142] [Show and tell]
Using WayBack machine, you can actually use non-iframeable websites. I got youtube.com working and I can even watch videos on it. Unfortunately, I cannot search because it changes the language.

**Comment by DustinBrett** [comment]
Could you post an example link and a use case?

**Reply by Pixel556** [reply]
https://web.archive.org/web/20180526000226/youtube.com Just visit this site on the Browser (not YOUR browser! the daedalOS browser.)
Internet Archive is iFrameable.

**Comment by DustinBrett** [comment]
I was thinking of adding Proxy support to the browser. So we could have various ways to go from a url to a page, such as internet archive or whatever else I can find and perhaps use to bypass CORS.

**Reply by hello-smile6** [reply]
@DustinBrett Why not use archive.org? You'd never even get a 404 error that way.

**Reply by DustinBrett** [reply]
It may be one in the list but it would not be the only option or default on. Anything online can have 404's and a large point of my project is its ability to be somewhat standalone, so I try not to rely on external sites that may not exist one day. I've been around long enough on the net to know nothing is certain to stay up.

**Reply by hello-smile6** [reply]
> It may be one in the list but it would not be the only option or default on. Anything online can have 404's and a large point of my project is its ability to be somewhat standalone, so I try not to rely on external sites that may not exist one day. I've been around long enough on the net to know nothing is certain to stay up.

Good point.
